test/test_win32_service.rb

```diff
@@ -10,14 +10,18 @@
 
 class TC_Win32_Service < Test::Unit::TestCase
   def self.startup
+    @@win_ver = Win32::Service.windows_version
     @@host = Socket.gethostname
-    @@service_name = 'stisvc'
-    @@elevated = Win32::Security.elevated_security?
+    @@service_name = @@win_ver < 6 ? 'Schedule' : 'LanmanServer'
+    # win32-security 0.3.1 crashes on 2003, so just assume elevated there
+    @@elevated = false
```

Comment:
The failure in tests triggered by the `win32-security` gem is:

```
Error: TC_Win32_Service
: Errno::EPERM: Operation not permitted - GetTokenInformation
C:/DOCUME~1/ADMINI~1/LOCALS~1/Temp/1/win32-service/.bundle/gems/ruby/2.1.0/gems/win32-security-0.3.1/lib/win32/security.rb:61:in `block in elevated_security?'
     58:             rl
     59:           )
     60:
  => 61:           raise SystemCallError.new("GetTokenInformation", FFI.errno) unless bool
     62:
     63:           result = te.read_ulong != 0
     64:         ensure
C:/DOCUME~1/ADMINI~1/LOCALS~1/Temp/1/win32-service/.bundle/gems/ruby/2.1.0/gems/win32-security-0.3.1/lib/win32/security.rb:41:in `initialize'
C:/DOCUME~1/ADMINI~1/LOCALS~1/Temp/1/win32-service/.bundle/gems/ruby/2.1.0/gems/win32-security-0.3.1/lib/win32/security.rb:41:in `new'
C:/DOCUME~1/ADMINI~1/LOCALS~1/Temp/1/win32-service/.bundle/gems/ruby/2.1.0/gems/win32-security-0.3.1/lib/win32/security.rb:41:in `elevated_security?'
C:/DOCUME~1/ADMINI~1/LOCALS~1/Temp/1/win32-service/test/test_win32_service.rb:19:in `startup'
================================================================================================================================================================
```

To summarize the failures without the change I've made

```
----------------------------------------------------------------------------------------------------------------------------------------------------------------

134 tests, 238 assertions, 0 failures, 7 errors, 0 pendings, 2 omissions, 1 notifications
95.4545% passed
----------------------------------------------------------------------------------------------------------------------------------------------------------------

23.12 tests/s, 41.06 assertions/s
```